articles/architecture-scenarios/implementation/b2c/b2c-profile-mgmt.md

```diff
@@ -69,31 +69,43 @@ Any information that will be used to customize Auth0 emails, such as information
 
 The Auth0 Management API, on the other hand, can be used to [directly change the password](/connections/database/password-change#directly-set-the-new-password) for a user identity defined using a Database Connection type. The Auth0 Management API can be used as part of any self-service profile management implementation, and is also used as part of any [Change Password page customization](/architecture-scenarios/implementation/b2c/b2c-branding#change-password-page-customization). 
 
 ## Account verification
 
-You’ll also want to make sure that you are working with a verified user account at all times, and make use of the mechanisms Auth0 provides. You should also consider regulatory compliance like [GDPR](https://eugdpr.org/) which has some very specific requirements when it comes to protecting all EU citizens from privacy and data breaches and guidance.  
+You’ll also want to make sure that you are working with a verified user account at all times, and make use of the mechanisms Auth0 provides. You should also consider regulatory compliance like [GDPR](https://eugdpr.org/), which has some very specific requirements when it comes to protecting all EU citizens from privacy and data breaches and guidance.  
 
-Auth0 provides out of box functionality for sending a [verification email](/email/custom#verification-email) to a user's email address as one way of verifying their account. By default, Auth0 is configured to automatically send verification emails for any database connections user identity created (e.g as part of [self sign up](/architecture-scenarios/implementation/b2c/b2c-provisioning#self-sign-up)). However, Auth0 also provides a [Management API endpoint](/api/v2#!/Tickets/post_email_verification) which can also be used to send verification emails (e.g. in cases where email addresses validation is not performed by a Social Provider upon user registration). 
+Auth0 provides out-of-box functionality for sending a [verification email](/email/custom#verification-email) to a user's email address as one way of verifying their account. By default, Auth0 is configured to automatically send verification emails for any database connections user identity created (e.g., as part of [self sign-up](/architecture-scenarios/implementation/b2c/b2c-provisioning#self-sign-up)). However, Auth0 also provides a [Management API endpoint](/api/v2#!/Tickets/post_email_verification) that can also be used to send verification emails (e.g., in cases where email address validation is not performed by a Social Provider upon user registration). 
```

Comment:
Just a head's up: Not sure what "By default, Auth0 is configured to automatically send verification emails for any database connections user identity created" means? Is it missing a word or a hyphen perhaps?

Reply:
Good spot @rkhoriander 👍 I've updated content with what I believe should be the correct text.